iFixit/react-commerce
# Add Sharp to Dependencies for Production.

## Overview

When running `next start` we were getting the warning that the optional sharp dependency should be added when running a production deployment. This should solve that error by adding `sharp` as a dependency to the project.

## CR
Super simple.

## QA
Easy to test.
1.) run `pnpm install:all`
2.) run `pnpm build`
3.) run `pnpm start`
4.) go to `localhost:3000/Parts`
5.) click on "Android Parts"
6.) a warning should show up in your terminal for using `sharp` if it isn't installed here (ie on main), and shouldn't if it is installed (ie on this branch).

Closes: #397
